networkx/algorithms/traversal/depth_first_search.py

```diff
@@ -231,26 +294,33 @@ def dfs_preorder_nodes(G, source=None):
     Examples
     --------
     >>> G = nx.Graph()
-    >>> G.add_path([0,1,2])
+    >>> G.add_path([0, 1, 2, 3, 4])
     >>> print(list(nx.dfs_preorder_nodes(G,0)))
+    [0, 1, 2, 3, 4]
+
+    >>> print(list(nx.dfs_preorder_nodes(G,0,2)))
```

Comment:
You don't need to have `print` here, just do `list(...)`, since it is part of an interactive shell.